arangod/Aql/Functions.cpp

```diff
@@ -7523,3 +7524,85 @@ AqlValue Functions::NotImplemented(ExpressionContext* expressionContext,
   registerError(expressionContext, "UNKNOWN", TRI_ERROR_NOT_IMPLEMENTED);
   return AqlValue(AqlValueHintNull());
 }
+
+AqlValue Functions::GetValidation(ExpressionContext* expressionContext,
+                              transaction::Methods* trx,
+                              VPackFunctionParameters const& parameters) {
+
+  static char const* AFN = "GET_SCHEMA"; // GET_VALIDATON("collectionName")
+
+  if(parameters.size() != 1) {
+    THROW_ARANGO_EXCEPTION_PARAMS(TRI_ERROR_QUERY_FUNCTION_ARGUMENT_NUMBER_MISMATCH, AFN);
+  }
+
+  std::string const collectionName = ::extractCollectionName(trx, parameters, 0);
+
+  if (collectionName.empty()){
+    THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_BAD_PARAMETER, "could not extract collection name from parameters");
+  }
+
+  auto logicalCollection = expressionContext->vocbase().lookupCollection(collectionName);
+  if(!logicalCollection) {
+    THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_ARANGO_DATA_SOURCE_NOT_FOUND, "could not find collection: " + collectionName);
+  }
+
+  VPackBuilder builder;
+  logicalCollection->validatorsToVelocyPack(builder);
+  auto slice = builder.slice();
+
+  if(!slice.isObject()) {
+    THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_BAD_PARAMETER, "collection has no validaiton object");
+    return AqlValue(AqlValueHintNull());
```

Comment:
Interestingly, there is also code to return `null`, but it currently is never reached due to the exception being thrown directly in front of it.
I suggest we remove throwing the exception and return `nul` simply.